### PR TITLE
v2022.08.0

### PR DESCRIPTION
**Pull request by marodrig**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by marodrig** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ requirements:
     - eofs
     - geocat-f2py
     - metpy
-    - numpy
+    - numpy
```
Comment (anissa111) [inline recipe/meta.yaml:38]: Probably shouldn't add trailing whitespace here

File: recipe/meta.yaml [hunk 2]
```diff
@@ -45,12 +45,16 @@ test:
     - pip
     - netcdf4
     - geocat-datafiles
+    - numba
```
Comment (anissa111) [inline recipe/meta.yaml:48]: We aren't actually using numba in geocat-comp (yet), I don't think we should add it to the `meta.yml` here.

**Review by anissa111 — CHANGES_REQUESTED** [review]

**Comment by anissa111** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 2 above)
Comment (anissa111) [reply recipe/meta.yaml:47]: Note for future developers: numba is needed by xskillscore

**Review by anissa111 — APPROVED** [review]

**Pull request merged**